docs/get-started/_index.md

```diff
@@ -1,36 +1,97 @@
 
-{{< cards >}}
-{{% card link="/get-started/installation/" %}}
-{{% manualcard link="https://discord.gg/viam" webm_src="/heart.webm" mp4_src="/heart.mp4" alt="A robot drawing a heart" %}}
+The Viam App provides the user interface for configuring your machine.
+It allows you to configure builtin drivers or drivers from the Viam Registry.
+`viam-server` then manages and runs the drivers for the configured resources.
+As you configure your machine, you can test each resource to confirm it is working as expected.
+
+{{<gif webm_src="/test.webm" mp4_src="/test.mp4" alt="Test a camera stream" max-width="700px" class="aligncenter">}}
+
+Viam is built to be extensible, allowing you to create your own resources for custom hardware or software and deploy your code using modules through the Viam Registry.
+
+On top of configuration, the Viam App also provides several higher-level functionalities for your machine, such as:
+
+- **Data Management**: Any data on your machine can be synced to the cloud, from where you can query it using SQL, MQL, or with code.
+- **Machine Learning**: Train machine learning models on collected data and deploy ML models to machines to enable them to intelligently see and interpret the world around them.
+- **Motion Planning**: Plan and move machine components.
+- **Simultaneous Localization And Mapping (SLAM)**: A machine can map its surroundings and find its position on a map.
+
+## Standardized APIs
+
+Each category of {{< glossary_tooltip term_id="resource" text="resource" >}} has a standardized API that you can access with an [SDK (software development kit)](/sdks/) in your preferred programming language.
+For example, you can send the same [`SetPower` command](/components/motor/#setpower) to any kind of motor, using any of the available SDKs.
+
+The standardized nature of the resource APIs, means that when you build machines, you can swap out components such as motors without changing any code.
+The only change needed is to your machine configuration.
+
+The Viam [Registry](/registry/) provides a marketplace for adding and sharing modules, all using these standard APIs.
 
-#### Community
+## Scale
 
-Have questions, or want to meet other people working on smart machines? Join us in the Community Discord!
+The Viam App does not limit you to managing just one machine.
+Viam's **fleet management** capabilities enable you to:
 
-{{% /manualcard %}}
+- Manage permissions within your organization and locations.
+- Manage software across your fleet, including deployment of code and machine learning models.
+- Configure, control, debug, and manage groups of machines in one go.
 
+## Support
+
+Viam supports devices running **any** 64-bit Linux OS or macOS.
+Here are some of the boards Viam supports:
+
+{{< board-carousel >}}
+<br>
+
+## Network flexibility
```

Comment:
This section puts a lot of emphasis on being able to have your machine work offline, but what I believe to be a very differentiating feature for Viam is only mentioned in passing: the ability to control and monitor your machines securely from anywhere with the programming language of your choice.  This should imo be highlighted clearly and distinctly.